### PR TITLE
Goerli testnet

### PR DESCRIPTION
**Pull request by dshulyak**
I wasn't able to run test suite using goerli testnet yet. Faucet seems to be stuck, i got a confirmation from it that funds were allocated. But according to etherescan nothing was transacted to any of my addresses.

Closes: https://github.com/status-im/status-go/issues/1413
This change includes geth update https://github.com/status-im/status-go/pull/1414

### CODE REVIEW

**Comment by dshulyak** [comment]
It looks like that our e2e transactions suite verifies only that transactions were sent. So it basically doesn't make any difference which network is used. Also, the suite gets stuck on synchronization with any network except our mock, StatusChain.

File: params/defaults.go [hunk 1]
```diff
@@ -39,6 +39,10 @@ const (
 	// allow us avoid syncing node.
 	RinkebyEthereumNetworkURL = "https://rinkeby.infura.io/nKmXgiFgc2KqtoQ8BCGJ"
 
+	// GoerliEthereumNetworkURL is an open RPC endpoint to Goerli network
+	// Other RPC endpoints are available here: http://goerli.blockscout.com/
+	GoerliEthereumNetworkURL = "http://goerli.blockscout.com/"
```
Comment (adambabik) [inline params/defaults.go:44]: So Infura does not support Goerli?

Comment (dshulyak) [reply params/defaults.go:44]: yeah, no rpc endpoint from them. they are going to setup one in the future. also they run one of the block signers for this poa network

**Comment by dshulyak** [comment]
So what about this change? Are there any concerns?

**Review by adambabik — APPROVED** [review]

**Pull request merged**